jojo.py

```diff
@@ -107,6 +107,172 @@ def open_json(json_file):
+    
+    #「%target%」箇所の置換
+    cmd = cmd.replace(f'%target%', '') if target is None else cmd.replace(f'%target%', f'\u0020{target}')
+
+    return mcr.command(cmd)
+
+
+def set_entity_data(mcr, types, X, Y, Z, invulnerable, gravity, tag, name):
+    '''
+    指定されたエンティティを作成します。
+
+    Parameter
+        mcr : MCRcon
+            Rconのサーバ情報
+        types : str
+            検索対象のオブジェクト名
+        X : int
+            エンティティのX座標
+        Y : int
+            エンティティのY座標
+        Z : int
+            エンティティのZ座標
+        invulnerable : bool
+            エンティティの不死身属性(True：不死身、False：定命)
+        gravity : bool
+            エンティティの重力属性(True：重力、False：無重力)
+        tag : str
+            タグ情報
+        name : str
+            Name情報
+    Return
+        コマンドの実行結果
+    '''
+    #コマンドの基本構文を生成
+    cmd = f'/summon %types% %X% %Y% %Z% {{%invulnerable%%gravity%%tag%%name%}}'
+    
+    #「%types%」箇所の置換
+    cmd = cmd.replace(f'%types%', '') if types is None else cmd.replace(f'%types%', f'{types}')
+    
+    #「%X%」箇所の置換
+    cmd = cmd.replace(f'%X%', '') if X is None else cmd.replace(f'%X%', f'{X}')
+    
+    #「%Y%」箇所の置換
+    cmd = cmd.replace(f'%Y%', '') if Y is None else cmd.replace(f'%Y%', f'{Y}')
+    
+    #「%Z%」箇所の置換
+    cmd = cmd.replace(f'%Z%', '') if Z is None else cmd.replace(f'%Z%', f'{Z}')
+    
+    #「%invulnerable%」箇所の置換
+    cmd = cmd.replace(f'%invulnerable%', '') if invulnerable is None else cmd.replace(f'%invulnerable%', f'Invulnerable:1,') if invulnerable else cmd.replace(f'%invulnerable%', f'Invulnerable:0,')
+ 
+    #「%gravity%」箇所の置換
+    cmd = cmd.replace(f'%gravity%', '') if gravity is None else cmd.replace(f'%gravity%', f'NoGravity:0,') if gravity else cmd.replace(f'%gravity%', f'NoGravity:1,')
+        
+    #「%tag%」箇所の置換
+    cmd = cmd.replace(f'%tag%', '') if tag is None else cmd.replace(f'%tag%', f'Tags:[{tag}],')
+
+    #「%name%」箇所の置換
+    cmd = cmd.replace(f'%name%', '') if name is None else cmd.replace(f'%name%', f'CustomName:\'{name}\'')
+    
+    return mcr.command(cmd)
+
+
+def edit_entity_tag_data(mcr, types, old_tag, new_tag, name):
+    '''
+    指定されたエンティティのタグ名を変更します。
+
+    Parameter
+        mcr : MCRcon
+            Rconのサーバ情報
+        types : str
+            検索対象のオブジェクト名
+        old_tag : str
+            変更前のタグ名
+        new_tag : str
+            変更後のタグ名
+        name : str
+            Name情報
+    Return
+        コマンドの実行結果(list型)
+    '''
+    #コマンドの基本構文を生成
+    cmd = f'/tag @e[limit=1,%types%%name%] %command% %tag%'    
+    
+    #「%types%」箇所の置換
+    cmd = cmd.replace(f'%types%', '') if types is None else cmd.replace(f'%types%', f'type={types},')
+
+    #「%name%」箇所の置換
+    cmd = cmd.replace(f'%name%', '') if name is None else cmd.replace(f'%name%', f'name={name}')
+    
+    #「%command%」、「%tag%」箇所を置換しコマンド実行
+    remove_resp = mcr.command(cmd.replace(f'%command%', 'remove').replace(f'%tag%', '') if old_tag is None else cmd.replace(f'%command%', 'remove').replace(f'%tag%', f'{old_tag}'))

```

Comment:
現状tagコマンドの実行結果を使うことは無いので返り値用の変数は不要です。

```suggestion
    mcr.command(cmd.replace(f'%command%', 'remove').replace(f'%tag%', '') if old_tag is None else cmd.replace(f'%command%', 'remove').replace(f'%tag%', f'{old_tag}'))

```

Reply:
該当関数はあくまでもタグ名を変更する用途で使われるため、mcr.commandの実行結果として返り値が存在するのでればそれを返却するのが妥当かと思われます。
tagコマンドの実行結果を使うことは無いかどうかは呼び出し元関数の話であり、該当関数を修正する必要はありません。